docs/DeveloperGuide.md

```diff
@@ -700,35 +700,110 @@ Given below are instructions to test the app manually.
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
 testers are expected to do more *exploratory* testing.
-
 </div>
 
-### Launch and shutdown
-
-1. Initial launch
-
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+### General
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+#### Launching TutorBuddy
+1. Download the jar file and copy into an empty folder
+1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+#### Saving Window Preferences
+1. Resize the window to an optimum size. Move the window to a different location. Close the window.
```

Comment:
Possibly want to indicate that we have a restriction on the maximum window size?

Reply:
+1

Reply:
I think that would be too much details?